core/modules/widgets/navigator.js

```diff
@@ -499,7 +499,8 @@ NavigatorWidget.prototype.handleImportTiddlersEvent = function(event) {
 	// Get the tiddlers
 	var tiddlers = $tw.utils.parseJSONSafe(event.param,[]);
 	// Get the current $:/Import tiddler
-	var importTitle = event.importTitle ? event.importTitle : IMPORT_TITLE,
+	var paramObject = event.paramObject || event,
```

Comment:
With this change, the code looks both at the toplevel of event or in event.paramObject. I could have changed it to only look in event.paramObject. With that approach, dropzone.js would also need to be changed to pass parameters in paramObject.